app/models/lesson.rb

```diff
@@ -8,8 +8,12 @@ class Lesson < ApplicationRecord
   belongs_to :author, class_name: 'User'
   enum :status, STATUSES
 
-  validates :title, presence: true
-  validates :description, presence: true
-  validates :video_link, presence: true
+  validates :title, presence: true, length: { maximum: 64, too_long: 'The maximum allowed number of character is 64' },
```

Comment:
Please, create constants for **64** and **600** as you use it more than once:

TITLE_MAX_SIZE = 64
...

length: { maximum: TITLE_MAX_SIZE , too_long: "The maximum allowed number of character is #{TITLE_MAX_SIZE }" },

etc.